backend/users-service/app/.prettierrc.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
nit: might be good to have same prettier settings as the FE

Reply:
> nit: might be good to have same prettier settings as the FE

Good point, can include that in the UI PR